compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_nested_layout_vector_distribution.mlir

```diff
@@ -33,11 +33,11 @@ builtin.module attributes { transform.with_named_sequence } {
   }
 }
 
-// CHECK: %[[IDS:.+]]:4 = affine.delinearize_index %{{.*}} into (%c1, %c1, %c4, %c8) : index, index, index, index
-// CHECK: %[[LANEY:.+]] = affine.apply #[[$MAP]]()[%[[IDS]]#2]
-// CHECK: %[[RD00:.+]] = vector.transfer_read %arg0[%[[LANEY:.+]], %[[IDS]]#3], {{.*}} : memref<32x32xf16>, vector<4x1xf16>
+// CHECK: %[[TIDS:.+]]:2 = iree_vector_ext.thread_ids %{{.*}}
```

Comment:
AFAICT you dropped the `thread_ids` and `subgroup_ids` ops (I can't find them in this PR at least). Is this leftover from before dropping those ops?

Reply:
yeah i have to update this test. Other tests are updated. This file is 1000 lines long so I need to spend some time cleaning it 😭

Reply:
fixed